backend/src/main/java/net/vidux/camhub/discovery/ViduxHelperRunner.java

```diff
@@ -0,0 +1,31 @@
+package net.vidux.camhub.discovery;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+class ViduxHelperRunner {
+  List<String> callViduxHelper() throws TimeoutException, IOException {
+    ProcessBuilder builder = new ProcessBuilder();
+    builder.command("vidux-helper", "system", "findHikvisionIPCameras");
```

Comment:
A `command` metódusnak átadott String tömböt szerintem kiemelném konstansként, amit itt csak át kell adni a command-nak.
Pl.: FIND_HIKVISION_IP_CAMERAS_COMMAND, vagy valami rövidebb elnevezés is lehet, de legyen beszédes.